test/integration/monitoring-alert/monitoring_alert_test.go

```diff
@@ -0,0 +1,131 @@
+		projectDescribe := gcloud.Runf(t, "projects describe %s", projectId)
+		projectNumber := projectDescribe.Get("projectNumber").String()
+		secretDescribe := gcloud.Runf(t, "secrets describe %s --project %s", "secret-1", projectId)
+		secretName := secretDescribe.Get("name").String()
+		assert.Equal(fmt.Sprintf("projects/%s/secrets/secret-1", projectNumber), secretName, "has expected name")
+
+		notificationChannelNames := secretT.GetJsonOutput("notification_channel_names").Array()
+		assert.Len(notificationChannelNames, 1)
+		notificationChannelName := notificationChannelNames[0].String()
+		monitoringChannel := gcloud.Runf(t, "beta monitoring channels list --project %s --filter name='%s'", projectId, notificationChannelName).Array()
+		assert.Len(monitoringChannel, 1)
+		assert.Equal(email_address, monitoringChannel[0].Get("labels.email_address").String())
+
+		monitoringAlerts := gcloud.Runf(t, "alpha monitoring policies list --project %s", projectId).Array()
+		assert.Len(monitoringAlerts, 1)
+		monitoringAlert := monitoringAlerts[0]
+		alertCondition := monitoringAlerts[0].Get("conditions").Array()
+		assert.Len(alertCondition, 1)
+		expectedFilter := fmt.Sprintf("protoPayload.methodName=\"google.cloud.secretmanager.v1.SecretManagerService.DestroySecretVersion\" AND protoPayload.resourceName : \"%s\"", secretT.GetJsonOutput("secret_names").Array()[0].String())
+		assert.Equal(expectedFilter, alertCondition[0].Get("conditionMatchedLog.filter").String())
+		notificationChannel := monitoringAlert.Get("notificationChannels").Array()[0].String()
+		assert.Equal(notificationChannelName, notificationChannel)
+		assert.Equal("WARNING", monitoringAlert.Get("severity").String())
+		assert.Equal("300s", monitoringAlert.Get("alertStrategy.notificationRateLimit.period").String())
+		assert.True(monitoringAlert.Get("enabled").Bool())
+
+		time.Sleep(5 * time.Minute)
+		gcloud.Runf(t, "secrets versions destroy %s/versions/1 -q", secretName)
+		utils.Poll(t, func() (bool, error) {
+			alertingLogs := gcloud.Runf(t, "logging read logName:\"projects/%s/logs/monitoring.googleapis.com\" --freshness=2m --project %s", projectId, projectId).Array()
+			for _, log := range alertingLogs {
+				expectedLogMessage := fmt.Sprintf("Log match condition fired for Audited Resource with {method=google.cloud.secretmanager.v1.SecretManagerService.DestroySecretVersion, project_id=%s, service=secretmanager.googleapis.com}", projectId)
+				logMessage := log.Get("labels.verbose_message").String()
+				expectedLogName := fmt.Sprintf("projects/%s/logs/monitoring.googleapis.com", projectId)
+				logName := log.Get("logName").String()
+				if strings.Contains(logMessage, expectedLogMessage) && strings.Contains(logName, expectedLogName) {
+					// Test succeded.
+					return false, nil
+				}
+			}
+			return true, errors.New("Alert wasn't fired correctly.")
+		}, 200, 10*time.Second)
+	})
+	secretT.Test()
+}
+
+func TestMonitoringAlertAllSecrets(t *testing.T) {
```

Comment:
maybe clarify the test name. I think this is to alert everyone specified, if more than 1 identity is provided.  
I thought initially that this would test if there are more than 1 secrets